### PR TITLE
Newest Version of Mood Quiz

### PR DESCRIPTION
**Pull request by joerossi215**

### CODE REVIEW

File: MoodQuiz2.0.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+
+	<div class="container text-center results hide">
+		<p id="results"></p>
+	</div>
+
+	<div class="container text-center bottom">
+		<button id="submit-btn" class="btn btn-primary btn-lg">Submit</button>
+
+		<button id="retake-btn" class="hide btn btn-primary btn-lg">Retake Quiz</button>
```
Comment (Taraque-A) [inline MoodQuiz2.0.html:35]: It's look perfect to me.

**Review by yellow-tulips — APPROVED** [review]
Looks good!

**Comment by farzanaisrat** [comment]
LGTM!

**Review by farzanaisrat — APPROVED** [review]

**Review by tasnhubaislam — APPROVED** [review]
Looks pretty good. Good Job!

File: MoodQuiz.html [hunk 2]
```diff
@@ -1,98 +1,43 @@
...
 <head>
-    <h1>Mood Quiz</h1>
-    <title>Mood Quiz</title>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Mood Quiz</title>
+	
+    <link rel="stylesheet" type="text/css" href="css/animate.css">
+	
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```
Comment (Taraque-A) [inline MoodQuiz.html:9]: Stylesheet looks perfect.

File: MoodQuiz2.0.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (Taraque-A) [inline MoodQuiz2.0.html:4]: Really good.

File: MoodQuiz2.0.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+	
+	<link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+	
+	<link rel="stylesheet" type="text/css" href="css/MoodQuiz2.0_CSS.css">
+</head>
+
+<body>
+
+	<div class="container text-center">
```
Comment (Taraque-A) [inline MoodQuiz2.0.html:18]: Mood quiz looks good.

File: MoodQuiz2.0.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+		<h1>Mood Quiz</h1>
+	</div>
+	<div class="container text-center">
+		<ul id="quiz" class="list-group">
+
+		</ul>
+	</div>
+
+
+	<div class="container text-center results hide">
```
Comment (Taraque-A) [inline MoodQuiz2.0.html:28]: Results looks nice.

**Comment by joerossi215** [comment]
Reviewers approved pull request

**Pull request merged**